tools/integration_tests/rapid_appends/appends_test.go

```diff
@@ -34,353 +33,252 @@ const (
 	unfinalizedObjectSize = 10 // Size in bytes of initial unfinalized Object.
 )
 
-////////////////////////////////////////////////////////////////////////
-// Tests
-////////////////////////////////////////////////////////////////////////
+// //////////////////////////////////////////////////////////////////////
+// Tests for the AppendsTestSuite
+// //////////////////////////////////////////////////////////////////////
+
+func (s *AppendsTestSuite) TestAppendSessionInvalidatedByAnotherClientUponTakeover() {
```

Comment:
Minimize unnecessary diff and stick to the convention in the other test packages.

```suggestion
func (t *AppendsTestSuite) TestAppendSessionInvalidatedByAnotherClientUponTakeover() {
```

Reply:
The same goes for all the other tests.